README.md

```diff
@@ -51,4 +51,4 @@ export DETRAY_BFIELD_FILE="${PWD}/odd-bfield_v0_9_0.cvf"
 
 Monitoring the propagation speed with the toy geometry
 
-<img src="https://gitlab.cern.ch/acts/detray-benchmark/-/raw/master/plots/array_data.png?ref_type=heads" width="500" height="500" /> 
\ No newline at end of file
+<img src="https://gitlab.cern.ch/acts/detray-benchmark/-/raw/master/plots/array_data.png?ref_type=heads" width="500" height="500" /> 
```

Comment:
Is this an added/removed newline character? What's going on with this line? 😕

Reply:
As you can compare, there is no change in the line. I also didnt touch this line so it is not on me

Reply:
And yet, when a computer does things that I didn't expect, I like to investigate. 😛

I'm 99% sure that it's a matter of having gained a newline character at the end of the line. So, sure...